homeassistant/components/script/__init__.py

```diff
@@ -335,6 +339,16 @@ async def async_turn_on(self, **kwargs):
         self.hass.async_create_task(coro)
```

Comment:
The result of this is never checked, and exceptions may be silently swallowed.
Should that be fixed?

Reply:
Yes

Reply:
How?